src/main/java/build/buildfarm/instance/queues/WorkerQueues.java

```diff
@@ -90,24 +90,21 @@ public boolean enqueueOperation(Operation operation, SetMultimap<String, String>
     return true;
   }
 
-  public boolean AddWorker(SetMultimap<String, String> provisions, MatchListener listener) {
+  public void AddWorker(SetMultimap<String, String> provisions, MatchListener listener) {
     try {
       WorkerQueue queue = MatchEligibleQueue(provisions);
       queue.workers.add(new Worker(provisions, listener));
     } catch (Exception e) {
-      return false;
     }
-    return true;
```

Comment:
I guess its okay to get rid of the bool if the callers aren't using it.. Might be better to make the callers check the return instead and log failures